Gemfile

```diff
@@ -12,7 +12,7 @@ gem 'httpclient', '~> 2.7.0'
 gem 'jquery-rails'
 gem 'mrt-ingest', '0.0.4'
 gem 'net-ldap', '~> 0.16'
-gem 'nokogiri', '~> 1.8', '>= 1.8.2'
+gem 'nokogiri', '~> 1.10'
```

Comment:
try not to peg gem versions unless you really need to. The Gemfile.lock should ensure that the correct versions are used on whatever machine you're deploying to